unittest/ofdm_phase_est_bw.sh

```diff
@@ -17,7 +17,7 @@ results=$(mktemp)
 fading_dir=$1
 # BER should be < 5% for this test
 ofdm_mod --in /dev/zero --testframes 300 --mode 2020 --ldpc -p 312 --verbose 0 | \
-cohpsk_ch - - -40 --Fs 8000 -f 10 --ssbfilt 1 --mpp --fading_dir $fading_dir | \
+ch - - --No -40 -f 10 --ssbfilt 1 --mpp --fading_dir $fading_dir | \
```

Comment:
These scripts were working before from what I remember, but I wonder if we need to be more specific with the paths to `ofdm_mod`, `ch`, etc. anyway. Thoughts?

Reply:
One thing I like about using script files is once you set up the `$PATH` you can just use the application name without any path wrangling.

Reply:
I thought the changes to PATH weren't there but I checked again and they're there. 👍